flex-squid/src/test/java/org/sonar/flex/grammar/directives/AttributesTest.java

```diff
@@ -31,7 +31,9 @@
   @Test
   public void test() {
     Assertions.assertThat(g.rule(FlexGrammar.ATTRIBUTES))
```

Comment:
@pynicolas I don't see the direct link with the fix of parse error with native and the additional test case on Attributes, could you explain please ?
Also if native is an acceptable keyword shouldn't it be tested in [IdentifierTest.java](https://github.com/SonarCommunity/sonar-flex/blob/master/flex-squid/src/test/java/org/sonar/flex/grammar/expressions/IdentifierTest.java#L33), WDYT ?